build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     testCompile("org.spockframework:spock-core:$spockVersion") { exclude group: 'org.codehaus.groovy' }
     testCompile "junit:junit:$junitVersion"
     testCompile "org.eclipse.jgit:org.eclipse.jgit:$jgitVersion"
+    testCompile "cglib:cglib-nodep:$cglibVersion"
```

Comment:
This is a dependency I needed for mocking the Executor class.
As the Executor class is not an interface, Spock requires the inclusion of cglib. 

Since it is not needed for the runtime of the plugin, I only add it on the test scope though.